polaris-react/src/components/LegacyCard/LegacyCard.scss

```diff
@@ -70,6 +70,10 @@
 .Section {
   padding: var(--p-space-4);
 
+  #{$se23} & {
+    padding: var(--p-space-2) var(--p-space-4);
+  }
+
```

Comment:
This is the same padding as below for `sm-up`. this makes the xs screens consistent